index.html

```diff
@@ -107,6 +107,92 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long freezeCount;
+      unsigned long pauseCount;
+      double totalFreezesDuration;
+      double totalPausesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>freezeCount</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Count the total number of video freezes experienced by this receiver.
+          It is a freeze if frame duration, which is time interval between two
+          consecutively rendered frames, is equal or exceeds
+          Max(3 * avg_frame_duration_ms, avg_frame_duration_ms + 150), where
+          avg_frame_duration_ms is linear average of durations of last 30 rendered
+          frames.
```

Comment:
Clarification: Does this mean that the value is undefined until 30 frames have arrived or that it will use the so-far average?

Reply:
And by undefined I meant 0

Reply:
it will be so-far average.